src/worldfactory.cpp

```diff
@@ -958,41 +1037,67 @@ int worldfactory::show_worldgen_tab_modselection( const catacurses::window &win,
             wputch( win, BORDER_COLOR, LINE_OXOX );
         }
 
+        // Draw filter
+        if( fpopup ) {
+            mvwprintz( win, filter_pos, c_cyan, "< " );
+            mvwprintz( win, filter_pos + point( filter_view_len + 2, 0 ), c_cyan, " >" );
+            // This call makes popup draw its string at position specified on popup initialization
+            std::string rendered_string = fpopup->query_string( /*loop=*/false, /*draw_only=*/true );
```

Comment:
`rendered_string` is unused. Why is it there?

Reply:
I thought it was there to fix some kind of 'unused return' warning, so left it as is